viixoo_app_engine/viixoo_core/docker/service.env

```
@@ -0,0 +1,2 @@
+TOKEN_ODOO="sdgsd324erfdfgwe4erew-fdgs5434fgr2@sda-dsfsdfsf#"
```

ponle a TOKEN_ODOO la palabra dummy- así 
```suggestion
TOKEN_ODOO="dummy-sdgsd324erfdfgwe4erew-fdgs5434fgr2@sda-dsfsdfsf#" # noqa
```
Y luego  # noqa para que el git guardian no nos diga que estamos poniendo claves en hardcode.